explore/src/main/scala/explore/proposal/package.scala

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.proposal
+
+import lucuma.core.util.NewType
+
+object PartnersDialogState extends NewType[Boolean]:
```

Comment:
Can we use the `Visible` newtype instead of defining a new one? Or at least define the newtype on top of `Visible`.

Reply:
Good idea